src/components/UI/Button.js

```diff
@@ -1,27 +1,21 @@
-import styles from './button.module.css'
+import styles, { container } from './button.module.css'
 
-export default function Button({ type, children, onClick }) {
-	let classes = styles.container
-
-	switch (type) {
-		case 'add':
-			classes += ` ${styles.add} ${styles.bg_blu}`
-			children = 'ADD'
-			break
-		case 'review':
-			classes += ` ${styles.review} ${styles.bg_blu}`
-			children = 'Review your tasks'
-			break
-		default:
-			children = 'Default'
-			break
-	}
+export default function Button({
+	children,
+	onClick,
+	background,
+	scaling,
+	weight
+}) {
+	if (!background) background = 'bg_blu'
+	if (!scaling) scaling = 'small'
+	if (!weight) weight = 'normal'
```

Comment:
Bit of JS trivia but you could also do your defaults like so, if you wished:
```suggestion
export default function Button({
	children,
	onClick,
	background = 'bg_blu',
	scaling = 'small',
	weight = 'normal'
}) {
```

Reply:
Thanks Jamie, completely forgot this was a thing!